modules/configuring_hyperthreading_for_a_cluster.adoc

```diff
@@ -64,16 +64,21 @@ $ cat /sys/devices/system/cpu/cpu0/topology/thread_siblings_list
 0-4
 ----
 
-. Apply the isolated and reserved CPUs in the `PerformanceProfile` YAML. For example, you could set logical cores CPU0 and CPU4 as `isolated`, and logical cores CPU1 and CPU5 as `reserved`. When you configure reserved and isolated CPUs, the infra containers in pods use the reserved CPUs and the application containers use the isolated CPUs.
+. Apply the isolated and reserved CPUs in the `PerformanceProfile` YAML. For example, you could set logical cores CPU0 and CPU4 as `isolated`, and logical cores CPU1 to CPU3 and CPU5 to CPU7 as `reserved`. When you configure reserved and isolated CPUs, the infra containers in pods use the reserved CPUs and the application containers use the isolated CPUs.
```

Comment:
```suggestion
. Apply the isolated and reserved CPUs in the `PerformanceProfile` YAML. For example, you can set logical cores CPU0 and CPU4 as `isolated`, and logical cores CPU1 to CPU3 and CPU5 to CPU7 as `reserved`. When you configure reserved and isolated CPUs, the infra containers in pods use the reserved CPUs and the application containers use the isolated CPUs.
```

IBM Style Guide: 

> Use to indicate ability. Use instead of "could". To indicate possibility, use "might". To indicate permission, use "may".

Reply:
done :)